src/trio/testing/_memory_streams.py

```diff
@@ -280,6 +280,15 @@ def put_eof(self) -> None:
         self._incoming.close()
 
 
+# These are required to a Sphinx bug. TODO: report the relevant bug (and remove this)
```

Comment:
Honestly not sure why this is happening. Maybe it's an attrs bug because that's the only relevant bump?